src/code/z_player_lib.c

```diff
@@ -2121,30 +2868,26 @@ void Player_DrawCircusLeadersMask(PlayState* play, Actor* actor) {
 
             gfx = &gfx[6];
         } else {
-            Player* player = (Player*)actor;
-            f32 temp_f0 = sqrtf(SQ(player->actor.velocity.x) + SQ(player->actor.velocity.z));
-            s16 phi_s0 = temp_f0 * 2000.0f;
+            f32 speedXZ = sqrtf(SQ(player->actor.velocity.x) + SQ(player->actor.velocity.z));
+            s16 phi_s0 = speedXZ * 2000.0f;
             f32 temp_f20;
 
-            bubbleVelocity.y = temp_f0 * 0.4f;
+            bubbleVelocity.y = speedXZ * 0.4f;
             bubbleAccel.y = -0.3f;
 
             if (phi_s0 > 0x3E80) {
                 phi_s0 = 0x3E80;
             }
 
             phi_s0 = player->actor.focus.rot.y + ((i != 0) ? phi_s0 : -phi_s0);
```

Comment:
This makes it look like `i` will only be 1 or 2 (eyes?), so it might be better to hardcode the array count?

Reply:
What do you mean?

Reply:
This code is for dealing with the tears coming from each eye of the mask, right? It's set up to work for precisely 2 of them (hence the ±).

Reply:
And the array size.

Reply:
Oh, there are two symbols that need to be in sync. I'll make a define for them then